client/components/Main/CreateNewSubnavDropdown.tsx

```diff
@@ -74,7 +80,8 @@ class CreateNewSubnavDropdownFn extends React.PureComponent<IProps> {
                     });
                 });
 
-            if (sortedTemplates.length > MORE_TEMPLATES_THRESHOLD) {
+            if (recentTemplates.length < MORE_TEMPLATES_THRESHOLD ||
+                sortedTemplates.length > MORE_TEMPLATES_THRESHOLD) {
```

Comment:
Why do you need to check if `recentTemplates` < 5? Regardless of the count of `recentTemplates` if `sortedTemplates` > 5 we need to show that button.

Reply:
We have to show that more button in both cases, like if recentTemplates < 5 or sortedTemplates > 5.
Let's assume we have `sortedTemplates ==4` and in `recentTemplates` we have `3` templates, but the user wants to use that one template, which is not in `recentTemplates`.

Reply:
Okay makes total sense, let's keep it then